src/app/index.ts

```diff
@@ -259,10 +260,11 @@ module.exports = yo.extend({
         
         if (this.project.isExcelFunctionsProject)
         {
-          this.fs.copyTpl(this.templatePath(`excel-custom-functions-preview/**`), this.destinationPath(), templateFills, null, { globOptions: { ignore: `**/*.placeholder` }});
+          // copy over custom functions files from Excel Custom Functions repo (this is the model we would like to have for all projects).
+          git.Clone("https://github.com/OfficeDev/Excel-Custom-Functions", this.destinationPath());

```

Comment:
This would ultimately make a .git directory in the destination. It would be good to know if we could make an option to clone but without creating the .git directory (thus just being a copy of the files themselves).

It could be considered a feature to have this as a git clone because you could potentially merge in fixes from origin later. But I could also see where someone might not want the .git stuff. I suppose they could just delete the .git folder afterwards if it isn't desired.